CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [Unreleased] 2024-07-29
```

Comment:
Put all the changes under a single [Unreleased] section without date part. The release process will manage the rest